packages/gallery-indicator/src/GalleryIndicator/GalleryIndicator.types.ts

```diff
@@ -0,0 +1,18 @@
+import { ComponentPropsWithRef } from 'react';
+
+import { DarkModeProps, LgIdProps } from '@leafygreen-ui/lib';
+
+export interface GalleryIndicatorProps
+  extends Omit<ComponentPropsWithRef<'ul'>, 'onClick'>,
+    DarkModeProps,
```

Comment:
Why omit the onClick?

Reply:
This shouldn't be here, i forgot to remove it

Reply:
sg